spec/rails_helper.rb

```diff
@@ -6,14 +6,6 @@
 require 'spec_helper'
 require 'rspec/rails'
 require 'capybara/rails'
-
-Shoulda::Matchers.configure do |config|
-  config.integrate do |with|
-    with.test_framework :rspec
-    with.library :rails
-  end
-end
```

Comment:
I re-installed RSpec because some of our configuration wasn't working (had to mess around with the spec_helper vs. rails_helper contents) ... didn't realize I removed this! Thanks for catching it.